GlotPress/gp-translation-helpers
# Don't double load non-async content

#### Problem

In #40 we switched discussions to be loaded upon first page load (to enable directly linking to comments). While it worked, it still loaded the tab contents via Ajax and overwrote it. 

#### Solution

This changes the architecture by implementing the inline loading via the class variable `$has_async_content`. Fixes #54.

#### Testing instructions
Load a discussion and observe the following:
- The discussion is properly loaded (including the comment form).
- The XHR request doesn't get the data for the discussion tab like here:
![Screenshot 2022-04-27 at 13 41 22](https://user-images.githubusercontent.com/203408/165510745-af0d42eb-212b-4284-943c-5cb6e65d82e3.png)
